src/Csv/CsvGenerator.fs

```diff
@@ -28,7 +29,8 @@ module internal CsvTypeBuilder =
       { TypeForTuple = typWithoutMeasure
         Property = ProvidedProperty(propertyName, typ, GetterCode = fun (Singleton row) -> Expr.TupleGet(row, index))
         Convert = fun rowVarExpr -> conv <@ TextConversions.AsString((%%rowVarExpr:string[]).[index]) @>
-        ConvertBack = fun rowVarExpr -> convBack (Expr.TupleGet(rowVarExpr, index)) } )
+        ConvertBack = fun rowVarExpr -> convBack (Expr.TupleGet(rowVarExpr, index))
+        Param = ProvidedParameter(propertyName, typ) } )
```

Comment:
You should adjust casing of constructor parameter here to camelCase. I think we do that in JsonProvider also